nivio-demo/docker-compose.yml

```diff
@@ -12,12 +12,21 @@ services:
   nivio:
     image: dedica/nivio:latest
     environment:
+      # to provide the environment variables from the command line
       SEED: ${SEED}
       DEMO: ${DEMO}
+
       # enable DEMO to have a preloaded landscape
       # DEMO: 1
+
       # SEED also loads data, but uses the sources from the webserver
-      # SEED: http://localhost:3000/example_env.yml
-    ports: 
-      - 8080:8080
+      # SEED: http://webserver:80/example_env.yml
 
+      # SEED to load data from a local file, enable also volumes
+      # SEED: /demo/dedica.yml
+    # volumes:
+      # - type: bind
+        # source: ../src/test/resources/example
+        # target: /demo
+    ports:
+      - 8080:8080
```

Comment:
All options work on my machine now. The nginx and nivio are in the same docker network (a default one), the nginx server must be called from the inside on port 80, before we had the port on the host (3000).

Reply:
👍